infra/kubernetes/main.tf

```diff
@@ -0,0 +1,81 @@
+// terraform config //
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+  }
+  # backend "s3" {
+  #   bucket  = "cncf-credits-infra-kubernetes-tfstate"
+  #   key     = "terraform.tfstate"
+  #   region  = "ap-southeast-2"
+  #   encrypt = true
+  # }
+}
+
+provider "aws" {
+  profile = "default"
+  region  = "ap-southeast-2"
```

Comment:
```suggestion
  region  = "us-east-1"
```

Reply:
I commented about this [elsewhere](https://github.com/cncf/credits/pull/2/files#r796133289).